widgetastic_manageiq/__init__.py

```diff
@@ -2322,3 +2327,58 @@ def fill(self, values):
 
     def read(self):
         return self.all_shortcuts
+
+
+class DynamicTable(VanillaTable):
```

Comment:
If it is true that this table is likely to not have the checkboxes, I am for having a separate class.